catalog/spatial/registry/registry-federation-admin-impl/src/test/resources/etc/registry/registry-custom-slots.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
Does this (and the other json file) need a copyright header? I don't actually know, but just want to be sure.